prisma/schema.prisma

```diff
@@ -41,6 +41,7 @@ model Resource {
   type          ResourceType
   categories    CategoriesOnResources[]
   configuration ResourceConfiguration
+  draft         Boolean                 @default(false)
```

Comment:
I suppose defaulting this to `true` would make prisma complain about updating existing records, while the default of `false` isn't quite a good reflection of reality.

I would actually love an alternative along the lines of

```diff
+ enum ResourcePublicationStatus {
+   DRAFT
+   READY_FOR_REVIEW
+   PUBLISHED
+ }


...
- published         Boolean  @default(false)
+ publicationStatus ResourcePublicationStatus                 @default(DRAFT)
```

because I think this would simplify the code in a lot of places, and also TS would force us to make sure that we're making the appropriate changes where "published" was being referenced before.